pkg/assetvalidator/noopvalidator.go

```diff
@@ -24,5 +24,17 @@ type NoopErrorAttributedAssetValidator struct {
 // Validate is a noop implementation, this validator will need to do something, and that is something that
 // varies company to company. For testing purposes, this will always throw an error
 func (*NoopErrorAttributedAssetValidator) Validate(ctx context.Context, attributedAsset domain.NexposeAttributedAssetVulnerabilities) error {
-	return errors.New("this will always throw an error")
+	errorsList := []error{errors.New("this will always throw an error")}
```

Comment:
@originalbchan This is why. Making all ValidationError and ValidationFailure types require a list makes no sense when implementing an individual check. The implementation details of the MultiValidator implementation are leaking into the AssetValidator interface. This is the kind of coupling that makes code harder to understand and/or modify in the future.

We can talk more about it when we can get some face time.